test/integration_test/lib/analysis/author/test_wh_table.py

```diff
@@ -0,0 +1,79 @@
+from lib.analysis.author.wh_table import *
+
+
+def test_generate_wh_table_authors():
+
+    nodelist_filename = './test/integration_test/data/graph_nodes.csv'
+    edgelist_filename = './test/integration_test/data/graph_edges.csv'
+    output_filename = './test/integration_test/data/wh_table_authors.csv'
+
+    req_file_data1 = """Height(h),Number of authors(i)
```

Comment:
`req_file_data1`, `req_file_data2` can be put in two different files in test/integration_test/data/ directory. You can port the `save_to_disk()` and `load_from_disk()` functions of [util.py](https://github.com/prasadtalasila/IRCLogParser/blob/master/lib/util.py#L240-L260) module from IRCLogParser project. They will come in very handy during testing.